<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eliane confecções</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">*{
border:none;
}
QLabel{
color: rgb(255, 255, 255);
}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>451</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
    background-color: rgb(85, 85, 255);
    color: white;
    border-top-left-radius: 5px;
    border-top-right-radius: 10px;
    padding: 5px 8px;
    margin: 1px;
}

QTabBar::tab:selected {
    background-color: rgb(65, 65, 230);
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Cadastro_costureiro">
     <attribute name="title">
      <string>Cadastro de Costureiros</string>
     </attribute>
     <widget class="QPushButton" name="btnExcluir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>411</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(85, 85, 255);
    color: rgb(255, 255, 255);
    border-radius: 5px;
    padding: 5px 10px;
    border: none;
}

QPushButton:hover {
  
	background-color: rgb(255, 1, 5);
}
</string>
      </property>
      <property name="text">
       <string>Excluir Costureiro</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabelaCostureiro">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>411</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
gridline-color: rgb(0, 0, 0);
selection-color: rgb(170, 170, 255);</string>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Endereço</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(85, 85, 255);
    color: rgb(255, 255, 255);
    border-radius: 5px;
    padding: 5px 10px;
    border: none;
}

QPushButton:hover {
    background-color: rgb(65, 65, 230); /* cor mais escura no hover */
}
</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border-radius: 5px;
    border: 1px solid #000; /* exemplo de borda preta */
    color: rgb(255, 255, 255); /* cor do texto */
    background-color: #333; /* opcional, só pra destacar o branco */
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Telefone</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="txtTelefone">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Digite o telefone, aqui ...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Endereço</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLineEdit" name="txtEndereco">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Digite o endeço, aqui ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="txtNome">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>Digite o nome do costureiro, aqui ...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnEditar">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>210</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgb(85, 85, 255);
    color: rgb(255, 255, 255);
    border-radius: 5px;
    padding: 5px 10px;
    border: none;
}

QPushButton:hover {
    background-color: rgb(65, 65, 230); /* cor mais escura no hover */
}
</string>
      </property>
      <property name="text">
       <string>Editar Costureiro</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>231</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">COLOR:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/icons/icons/logo_empresa_200px.png&quot;/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Soluções em sistemas &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;CONTATO:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;(11)92005-1133&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons_.qrc"/>
 </resources>
 <connections/>
</ui>
